.travis.yml

```diff
@@ -5,11 +5,6 @@ before_install:
 bundler_args: --without development
 language: ruby
 rvm:
-  - 1.8.7
```

Comment:
Can you please remove the changes to CI from the pull request?

Reply:
Done!